examples/typescript/platform-ref-aws/crossplane.ts

```diff
@@ -0,0 +1,77 @@
+import { App, Chart } from 'cdk8s';
+
+//import * as config from './crs/configuration.meta.pkg.crossplane.io';
+//import { Configuration } from 'crossplane-cdk';
+import { Configuration } from '../../../src';
+
+import * as xrds from './crs/xrd.apiextensions.crossplane.io';
+import * as comp from './crs/composition.apiextensions.crossplane.io';
+
+export function configYaml(crossplanePackage: App): Chart {
+
+    const yaml = new Chart(crossplanePackage, 'crossplane');
+    /*
+    config.platformRefAws(yaml, 'platform-ref-aws')
+    */
+
+    const config = new Configuration(yaml, 'platform-ref-aws', {
+        name: 'platform-ref-aws',
+        company: 'Upbound',
+        maintainer: 'Jared Watts <jared@upbound.io>',
+        keywords: ['aws', 'cloud-native', 'kubernetes', 'example', 'platform', 'reference'],
+        source: 'github.com/upbound/platform-ref-aws',
+        license: 'Apache-2.0',
+        descriptionShort: 'The AWS reference platform for Kubernetes and Data Services.\n',
+        description: 'This reference platform Configuration for Kubernetes and Data Services\nis a starting point to build, run, and operate your own internal cloud\nplatform and offer a self-service console and API to your internal teams.\n',
+        readme: 'This reference platform `Configuration` for Kubernetes and Data Services\nis a starting point to build, run, and operate your own internal cloud\nplatform and offer a self-service console and API to your internal teams.\nIt provides platform APIs to provision fully configured EKS clusters,\nwith secure networking, and stateful cloud services (RDS) designed to\nsecurely connect to the nodes in each EKS cluster -- all composed using\ncloud service primitives from the [Crossplane AWS\nProvider](https://doc.crds.dev/github.com/crossplane/provider-aws). App\ndeployments can securely connect to the infrastructure they need using\nsecrets distributed directly to the app namespace.\n\n[Quickstart\nGuide](https://github.com/upbound/platform-ref-aws/#quick-start)\n\n[APIs in this\nConfiguration](https://github.com/upbound/platform-ref-aws/#apis-in-this-configuration)\n\nTo learn more checkout the [GitHub\nrepo](https://github.com/upbound/platform-ref-aws/) that you can copy and\ncustomize to meet the exact needs of your organization!\n',
```

Comment:
File?

Reply:
Good idea, similar to how cdk8s okie handles config map from disk